README.md

```diff
@@ -0,0 +1,53 @@
+# Generic Config Importer
+
+A generic config importer which makes an API call to configure services running in a Docker swarm.
+
+## Where can this be used?
+
+- In a Docker swarm
+- Anywhere you're making a POST request to an API to import configs
+
+## Requirements
+
+- npm
+- Docker
+
+## Usage
+
+To use the config importer, you have to specify a docker-compose file. The compose file must look as follows, with `SERVICE` referring to the service to be configured.
+
+```yml
+version: '3.9'
+
+services:
+  <service to configure>-config-importer:
+    image: jembi/config-importer:<version-tag>
+    environment:
+      SERVICE_NAME: <service name>
+      SERVICE_API_PORT: <service port>
+      API_USERNAME: <service username>
+      API_PASSWORD: <service password>
+      JS_REPORT_SSL: <true/false>
```

Comment:
Do we need to call out JS_REPORT specifically here? Can't we drop that prefix?

Reply:
Good spot, thanks @rcrichton, it's been changed in [b00a448](https://github.com/jembi/api-config-importer/pull/1/commits/b00a448c8c05c63165834769887b7ce2e983d3b1)